<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TP M1GIG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="myOpenGLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>15</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="importAction"/>
    <addaction name="exportAction"/>
   </widget>
   <widget class="QMenu" name="menu_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionCommandes"/>
    <addaction name="action_propos"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_propos"/>
  </widget>
  <action name="settingsAction">
   <property name="text">
    <string>Paramètres</string>
   </property>
  </action>
  <action name="importAction">
   <property name="text">
    <string>Importer...</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>Exporter...</string>
   </property>
  </action>
  <action name="actionCommandes">
   <property name="text">
    <string>Commandes</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">myopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
